docs/src/staking/stake-accounts.md

```diff
@@ -76,8 +76,21 @@ with token balances of your choosing.
 The same stake and withdraw authorities can be assigned to multiple
 stake accounts.
 
-Two stake accounts that are not delegated and that have the same authorities
-and lockup can be merged into a single resulting stake account.
+#### Merging stake accounts
+Two stake accounts that have the same authorities and lockup can be merged into
+a single resulting stake account. A merge is possible between two stakes in the
+following states with no additional conditions:
+
+* two fully deactivated stakes
+* an inactive stake into a fully activating stake
```

Comment:
Activating stake accounts can only be merged during the activation epoch `E`.  If global delegation is congested and the specified account is partial delegated in `E+N` (that is, where `N != 0`), the account is ineligible for merge until it has fully activated.

I think "fully" may also confuse in the "activating" case.  While there is a sense of "partially" active stake in the globally congested state, there's no way to "partially" delegate a stake.  I've found it much more succinct to communicate this in terms of the "activation epoch" instead

Reply:
That's a good point.  I wasn't sure how to convey the concept of  "purely activating" stake, but since the "transient" stake state is mentioned elsewhere.  So instead of "fully activating stake", would you say here just "activating stake"?

Reply:
I'd probably go with "into an activating stake during its activation epoch" for this one